### ISSUE TITLE
Include utf8 characters in paths on CI

### ISSUE THREAD

**Issue by zanderso** [flutter/flutter#111633]
On CI, we deliberately include spaces in paths to ensure our tools are resilient to that situation. We also need to ensure that our tools are resilient when paths contain utf8 characters, for example 'Á'.

This issue is a request to set up some of the tooling and integration tests on the framework tree, and some of the engine builders to run under paths with characters like 'Á' in the path, and to ensure that we are testing with spaces in paths in more places.

**Comment by dnfield** [comment]
See https://flutter.googlesource.com/recipes/+/df036d82f2eaccb601c57ed408a2396e1d7e6b01/recipes/flutter/flutter.py#37 - we could just add a non-ascii character there and that would cover it for the framework.  

We may want to do the same for when we create temporary directories in tool tests. 

We don't seem to do this on engine checkouts but could.

**Comment by sealesj** [comment]
I opened this issue in the meantime after conversation about the change needing to land in the third party project. Would it be a reasonable next step to disable the failing test temporarily? https://github.com/flutter/flutter/issues/112819

**Comment by sealesj** [comment]
@zanderso Yes I will look back into this. I had re-triggered led with result https://chromium-swarm.appspot.com/task?id=5ddb8355a5c77010 test for Windows using this recipe patch https://flutter-review.googlesource.com/c/recipes/+/34140 but see encountered old-style '/ssl/certs/ca-bundle.crt' and still see a failure

**Comment by sealesj** [comment]
The above test was failing due to another path issue addressed by this fix https://github.com/flutter/engine/pull/36908

**Comment by sealesj** [comment]
Completed with https://flutter-review.googlesource.com/c/recipes/+/34140

**Comment by zanderso** [comment]
Thanks, @sealesj!
